float64 time
uint8 type  # from PrimitiveType

MoveToPoseParam move_to_pose_param
ConstantVelocityParam constant_velocity_param
AdmittanceMotionParam admittance_motion_param

---
float64 time
uint8 status
